ui/src/app/pages/data/data-set/dataset-page.tsx

```diff
@@ -746,6 +746,14 @@ const apiEnumToPrePackageConceptSets = (
         re.add(PrepackagedConceptSet.FITBITACTIVITY);
         break;
       }
+      case PrePackagedConceptSetEnum.FITBITSLEEPDAILYSUMMARY: {
+        re.add(PrepackagedConceptSet.FITBITSLEEPDAILYSUMMARY);
+        break;
+      }
+      case PrePackagedConceptSetEnum.FITBITSLEEPLEVEL: {
+        re.add(PrepackagedConceptSet.FITBITSLEEPLEVEL);
+        break;
+      }
```

Comment:
Can we make this more bullet proof? You should be able to do something like this so this doesn't bite us everytime we add a new prepackaged type:
```suggestion
      case PrePackagedConceptSetEnum.BOTH: {
        re.add(PrepackagedConceptSet.PERSON);
        re.add(PrepackagedConceptSet.SURVEYS);
        break;
      }
      case PrePackagedConceptSetEnum.SURVEY: {
        re.add(PrepackagedConceptSet.SURVEYS);
        break;
      }
      case PrePackagedConceptSetEnum.NONE: {
        break;
      }
      default: {
        re.add(PrepackagedConceptSet[pre]);
        break;
      }
```

Reply:
Yeah, good call. I had to add a reverse enum to get the correct value but definitely cleaner and better going forward this way.

Reply:
Realized we basically had the same issue of needing to update with the static reverse enum. Just pushed a change to build the reverse enum based on the current enum properties.